service/docs/source/overview.rst

```diff
@@ -1,14 +1,758 @@
+components of both the Service and the Runtime are available via environment
+modules.  These components can be loaded on the command-line via:
+
+.. code-block:: bash
+
+    $ module load geopm
+
+To list all of the available GEOPM versions issue:
+
+.. code-block:: bash
+
+    $ module avail geopm
+
+General Users (non-ALCF)
+""""""""""""""""""""""""
+
+For users who are running one of our supported Linux distributions, see the
+following for installation instructions: :doc:`install`.  The GEOPM Runtime
+must be built from source and installed manually if desired.  For instructions
+on that process see: :doc:`devel`.
+
+.. note::
+
+    For usage with GPUs, please use the level-zero enabled RPMs here:
+
+    * SLES 15.3 (`v2.0.2 <https://download.opensuse.org/repositories/home:/geopm:/release-v2.0-supplementary/15.3/x86_64/>`__, `latest <https://download.opensuse.org/repositories/home:/geopm:/supplementary/15.3/x86_64/>`__)
+    * SLES 15.4 (`v2.0.2 <https://download.opensuse.org/repositories/home:/geopm:/release-v2.0-supplementary/15.4/x86_64/>`__, `latest <https://download.opensuse.org/repositories/home:/geopm:/supplementary/15.4/x86_64/>`__)
+    * CentOS 8 (`v2.0.2 <https://download.opensuse.org/repositories/home:/geopm:/release-v2.0-supplementary/CentOS_8/x86_64/>`__, `latest <https://download.opensuse.org/repositories/home:/geopm:/supplementary/CentOS_8/x86_64/>`__)
+
+Admin Configuration
+"""""""""""""""""""
+
+After the Service has been installed, it must be configured properly before
+non-root users will be able to leverage it.
+
+To grant permissions to **all** non-root users to be able to use **all** of the
+features provided by the Service issue the following:
+
+.. code-block:: bash
+
+    $ sudo geopmaccess -a | sudo geopmaccess -w
+    $ sudo geopmaccess -ac | sudo geopmaccess -wc
+
+These commands will create access lists in the system location that the Service
+will use to determine user privilege. More information on access list
+configuration can be found on the following pages: :doc:`admin` and
+:doc:`geopmaccess.1`.
+
+----
+
+|:card_file_box:| Understand the Platform Topology
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _topo-diagram:
+.. figure:: https://geopm.github.io/images/platform-topo-diagram.svg
+    :alt: Topology Encapsulation Diagram
+    :align: center
+
+The various layers of hardware in a system are described as *domains* in GEOPM
+terminology.  GEOPM has support for the following domains:
+
+* Board
+* Package
+* Core (physical)
+* CPU (Linux logical)
+* Memory
+* Package Integrated Memory
+* NIC
+* Package Integrated NIC
+* GPU
+* Package Integrated GPU
+* GPU Chip
+
+For more information on the domain types, see: :ref:`geopm_topo.3:Domain Types`.
+
+Code Examples
+"""""""""""""
+
+All of the code examples require linking against ``libgeopmd`` for C/C++.  The
+Python examples requires that your ``PYTHONPATH`` contains the ``geopmdpy``
+module.
```

Comment:
I think libgeopmd needs to be on the library path. Might matter if the user attempts to use https://pypi.org/project/geopmdpy/ without also installing libgeopmd.